src/main/js/controls/Scatter/helpers/helpers.js

```diff
@@ -209,7 +209,7 @@ const processDataPoints = (graphConfig, dataTarget) => {
     };
     // Update the interpolation type
     dataTarget.interpolationType = getInterpolationType(dataTarget.type);
-
+    dataTarget.showShapes = true;
```

Comment:
Why we are setting to true for scatter graph?

Reply:
Anyway for scatter graph shoeShapes will be always true 

> As I'm looking at the dev build and the screenshots, it looks like I need to be changing configuration values to test and verify this PR.
> 
> When that's the case, could you please give me some instructions as to what files and line numbers I should be looking at to make those changes?

@jeremyfuksa just to make your work easy i made those changes my self in this branch. You can just pull those changes and see the examples. Here is the [link](https://github.com/cerner/carbon-graphs/pull/297/files#diff-c7742c155a5bfded08c393dc7a4a2d40R43) of code changes i made in data.js. Please go through Line/simple, Line/y2 axis and Line/No data to see those examples.

Reply:
so is this line needed or is it going to be removed since it was only used for testing purposes?

Reply:
Sorry for the miscommunication . I did added this line on purpose while coding for this issue which i realized lately. The real reason for this is, showShapes(if true or false) does not effect anything in Scatter graph as it did in line graph. Since these graph share some same methods to generate shape in y-axis and legend, we need to adjust some things accordingly so that it should not effect current functionality and also satisfy the logic which we have. In order to that i did kept showShapes as true in scatter graph as default thing internally.

**Note:** Since there is no showShapes property for scatter graph, i am just using it internally to make things right when generating shapes in y-axis and legend.